routes/controllers/channel.controller.js

```diff
@@ -79,3 +79,28 @@ exports.createChannel = async (req, res, next) => {
     next(createError(500, ERR_MSG.SERVER_ERR));
   }
 };
+
+exports.getUserType = async (req, res, next) => {
+  try {
+    const { channelId, userId } = req.params;
+    const { audience } = await Channel.findById(channelId);
+    const audienceIdList = audience.map((user) => user._id.toString());
+    const isAudience = audienceIdList.find((audience) => audience === userId);
```

Comment:
```suggestion
    const isAudience = audienceIdList.some((audience) => audience === userId);
```

find는 찾은 `audience`(혹은 못 찾았으면 `undefined`)를 반환합니다. 따라서 find를 쓸거면 변수명은 `isAudience` 같은 boolean 형태의 변수명이 아니라 `audience`가 맞다고 생각하고 그에 따라서 밑에

```javascript
// data: isAudience ? 'audience' : 'player', 
data: audience != null ? 'audience' : 'player',
```

가 더 맞다고 생각합니다.

근데 굳이 audience를 찾아서 어떤 값으로 활용할 것이 아니기 때문에, 이 경우에는 boolean 값의 `isAudience`가 더 적합한 값이라고 생각하고 따라서`find` 메소드를 `some`으로 바꾸는 것이 좋을 것 같네요.